stable/traefik/values.yaml

```diff
@@ -259,25 +259,18 @@ acme:
 dashboard:
   enabled: false
   domain: traefik.example.com
-  service: {}
-    # annotations:
-    #   key: value
-  ingress: {}
-    # annotations:
-    #   key: value
-    # labels:
-    #   key: value
+  service:
+    annotations: {}
```

Comment:
Any particular reason that annotations should not be commented out, and the service key be the same as the original?

Reply:
Functionally there's no difference, I just wanted to make it clear that service. annotations now accepted yaml and not a key=>value map.  I'm fine with just changing the existing comments to `# annotations: {}` if that's more inline with your expectations.